### PR TITLE
Refactoring for consistency

### PR DESCRIPTION
**Pull request by JordanGibson**
Refactored pre-existing changes made by JamesMatchett to keep consistency with fixedWidth and characterCount.

### CODE REVIEW

File: Utilities/ConsoleTable.cs [hunk 1]
```diff
@@ -97,14 +97,14 @@ private static void WriteTableEntry(string key, string value)
...
         {
-            return (Word.Length + charactersUsedSoFar < maxWidth - 2);
+            return (word.Length + characterCount < fixedWidth);
         }
 
-        private static int NumberOfCharsLeftOnLine(int maxWidth, int charactersUsedSoFar)
+        private static int NumberOfCharsLeftOnLine(int characterCount)
         {
-            return (maxWidth - 2) - charactersUsedSoFar;
+            return fixedWidth - characterCount;
```
Comment (JamesMatchett) [inline Utilities/ConsoleTable.cs:107]: Does the -2 come into the fixedWidth here? just want to make sure all is working as normal

Comment (JordanGibson) [reply Utilities/ConsoleTable.cs:107]: Yes. MaxWidth is only used in the definition of rightSpacing. Functionality has remained as it was in #4 with the fix of #5

Comment (JamesMatchett) [reply Utilities/ConsoleTable.cs:107]: Righeto, as long as you're sure functionality will be as normal with this change, I'll merge now as I have no further questions.

**Comment by JordanGibson** [comment]
No functionality changes have been made as part of this commit.

**Pull request merged**